pep-0723.rst

```diff
@@ -284,27 +261,20 @@ installed in the user's environment.
 The risk here is part of the functionality of the tool being used to run the
 script, and as such should already be addressed by the tool itself. The only
 additional risk introduced by this PEP is if an untrusted script with a
-embedded metadata is run, when a potentially malicious dependency might be
-installed.
+embedded metadata is run, when a potentially malicious dependency or transitive
+dependency might be installed.
 
 This risk is addressed by the normal good practice of reviewing code
-before running it. Additionally, tools may be able to provide locking
-functionality when configured by their ``[tool]`` sub-table to, for example,
-add the resolution result as managed metadata somewhere in the script (this
-is what Go's ``gorun`` can do).
+before running it. Additionally, tools may be able to provide
+`locking functionality <723-tool-configuration_>`__ to ameliorate this risk.
 
 
 How to Teach This
 =================
 
-Since the format chosen is the same as the official metadata standard, we can
-have a page that describes how to embed the metadata in scripts and to learn
-about metadata itself direct users to the living document that describes
-`project metadata <pyproject metadata_>`_.
-
-We will document that the name and version fields in the ``[project]`` table
-may be elided for simplicity. Additionally, we will have guidance explaining
-that single-file scripts cannot (yet) be built into a wheel via standard means.
+We can have a page that describes how to embed the metadata in scripts. This
+will come naturally to many users because the allowed fields happen to match
```

Comment:
I hestitate to define what will come naturally -- Python's users globally are a heterogeneous set, who we don't know! Many uses in e.g. data science are by people who aren't expereienced programmers, etc.

I think the first line here "We can have..." isn't needed, as any proposal implemented will inevitably have documentation.

PEP-616 and PEP-655 have good how to teach this sections -- I think the major challenges with teaching here are the novel comment syntax (metadata blocks), and then the meaning assigned to those. This PEP should probably reccomend adding a descriptive guide to the PPUG at the least, probably also mentioning that the structed blocks only have meaning when used with tools that understand them, and that the core Python interpreter doesn't use them at all.